libnopegl/src/nopegl.h.in

```diff
@@ -177,8 +177,6 @@ NGL_API char *ngl_scene_dot(const struct ngl_scene *scene);
  */
 NGL_API void ngl_scene_unrefp(struct ngl_scene **sp);
 
-#define ngl_scene_freep ngl_scene_unrefp
```

Comment:
I would remove the function directly in the previous commit. IHMO, there is no point in keeping the intermediate state and that would simplify the history log.

Reply:
done